package.json

```diff
@@ -4,6 +4,7 @@
     "dev": "next dev",
     "build": "next build",
     "start": "next start -- --port $PORT",
+    "startt": "next start -- --port 3001",
```

Comment:
Maybe omit this

Reply:
instead of having an extra script, try doing the `export PORT=3001`